repo/packages/K/kubernetes/1/config.json

```diff
@@ -81,6 +81,15 @@
           "minimum": 0,
           "type": "integer"
         },
+        "resource-roles": {
```

Comment:
We need the role for the pods. That's what @s-urbaniak is working on in https://github.com/mesosphere/kubernetes-mesos/issues/482.

I fact, I would not even allow the user to customize the "apserver-resource-roles" at all. I don't see a good usecase for that. Users will misuse it as described moving the apiserver into the public internet.